### PR TITLE
PENG-2045 Fixed cluster_name presentation in multi-tenancy mode

### PR DESCRIPTION
**Pull request by dusktreader**
* Now, the user will see "cluster_name" when they view job_submissions instead of "client_id"
* When run in "multi-tenancy" mode, the organization_id is removed from the cluster_name when rendered

### CODE REVIEW

File: jobbergate-cli/jobbergate_cli/auth.py [hunk 1]
```diff
@@ -94,6 +94,29 @@ def validate_token_and_extract_identity(token_set: TokenSet) -> IdentityData:
...
+        try:
+            org_dict = token_data["organization"]
+            JobbergateCliError.require_condition(
+                len(org_dict) == 1,
+                message="Did not find exactly one organization in token payload",
+            )
+            identity.organization_id = list(org_dict.keys())[0]
+        except Exception as err:
+            raise Abort(
+                """
```
Comment (fschuch) [inline jobbergate-cli/jobbergate_cli/auth.py:108]: The error message could be a bit more specific, so we can quickly identify the a possible problem when providing support to end users

File: jobbergate-cli/jobbergate_cli/requests.py [hunk 2]
```diff
@@ -250,7 +250,7 @@ def make_request(
     if response_model_cls is None:
         return data
 
-    logger.debug("Validating response data with ResponseModel")
+    logger.debug(f"Validating response data with {response_model_cls}")
```
Comment (fschuch) [inline jobbergate-cli/jobbergate_cli/requests.py:253]: Good catch!

File: jobbergate-cli/poetry.lock [hunk 3]
```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.1 and should not be changed by hand.
```
Comment (fschuch) [inline jobbergate-cli/poetry.lock:1]: I see no change in dependencies on pyproject.toml. Could you revert the changes in this file?

File: jobbergate-cli/jobbergate_cli/subapps/pagination.py [hunk 4]
```diff
@@ -14,8 +17,10 @@ def handle_pagination(
     abort_message: str = "There was an error communicating with the API",
     params: Optional[Dict[str, Any]] = None,
     title: str = "Results List",
-    style_mapper: StyleMapper = None,
-    hidden_fields: Optional[List[str]] = None,
+    style_mapper: StyleMapper | None = None,
+    hidden_fields: List[str] | None = None,
+    nested_response_model_cls: Type[pydantic.BaseModel] | None = None,
```
Comment (fschuch) [inline jobbergate-cli/jobbergate_cli/subapps/pagination.py:22]: Nice touch. That was indeed missing.

**Review by fschuch — CHANGES_REQUESTED** [review]
Noce work. I left a few comments.

File: jobbergate-cli/jobbergate_cli/auth.py (hunk 1 above)
Comment (dusktreader) [reply jobbergate-cli/jobbergate_cli/auth.py:108]: We don't want to give the user too much information so that it's confusing or troubling. If the command is run in verbose mode, you will see the log_message below displayed. Additionally, Sentry should catch this error and provide more context for us as well.

File: jobbergate-cli/poetry.lock (hunk 3 above)
Comment (dusktreader) [reply jobbergate-cli/poetry.lock]: Rebased, removed, and force-pushed in commit a5b7946.

File: jobbergate-cli/jobbergate_cli/auth.py (hunk 1 above)
Comment (fschuch) [reply jobbergate-cli/jobbergate_cli/auth.py:108]: Fair enough

**Review by fschuch — APPROVED** [review]
